client/components/suggestion/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
```

Comment:
Suggestion is kinda broad. Let's be more specific and rename this component to something like suggested-address instead